### PR TITLE
place holder for kubeadm upgrade

### PR DESCRIPTION
**Pull request by RA489**
Place holder for kubeadm upgrade  of [kubernetes/kubeadm/issues/1422](https://github.com/kubernetes/kubeadm/issues/1422)

/assign @neolit123

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+- All containers are restarted after upgrade, because the container spec hash value is changed.
+- You can upgrade only from one minor version to the next minor version.
+  That is, you cannot skip versions when you upgrade.
+  For example, you can upgrade only from 1.10 to 1.11, not from 1.9 to 1.11.
+
+{{< warning >}}
+The command `join --experimental-control-plane` is known to fail on single node clusters created with kubeadm v1.12 and then upgraded to v1.13.x.
+This will be fixed when graduating the `join --control-plane` workflow from alpha to beta.
+A possible workaround is described [here](https://github.com/kubernetes/kubeadm/issues/1269#issuecomment-441116249).
+{{</ warning >}}
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: /assign @fabriziopandini 
we technically don't need the above warning anymore for 1.13->1.14?
am i correct?

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    [patchnode] Uploading the CRI Socket information "/var/run/dockershim.sock" to the Node API object "ip-10-0-0-7" as an annotation
+    [bootstraptoken] configured RBAC rules to allow Node Bootstrap tokens to post CSRs in order for nodes to get long term certificate credentials
+    [bootstraptoken] configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node Bootstrap Token
+    [bootstraptoken] configured RBAC rules to allow certificate rotation for all node client certificates in the cluster
+    [addons] Applied essential addon: CoreDNS
+    [addons] Applied essential addon: kube-proxy
+
+    [upgrade/successful] SUCCESS! Your cluster was upgraded to "v1.14.0". Enjoy!
+
+    [upgrade/kubelet] Now that your control plane is upgraded, please proceed with upgrading your kubelets if you haven't already done so.
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: we might have to update the output once done testing. but this is good for now.

**Comment by neolit123** [comment]
@kubernetes/sig-cluster-lifecycle-pr-reviews

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md (hunk 1 above)
Comment (timothysc) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: I like the warning for folks who might be concerned.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+    {{% /tab %}}
+    {{< /tabs >}}
+
+## Upgrade the control plane node
+
+1.  On your control plane node, upgrade kubeadm:
+
+    {{< tabs name="k8s_install_kubeadm" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.14.x-00 with the latest patch version
```
Comment (timothysc) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: that -00 versioning doesn't look right to me.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+    [bootstraptoken] configured RBAC rules to allow certificate rotation for all node client certificates in the cluster
+    [addons] Applied essential addon: CoreDNS
+    [addons] Applied essential addon: kube-proxy
+
+    [upgrade/successful] SUCCESS! Your cluster was upgraded to "v1.14.0". Enjoy!
+
+    [upgrade/kubelet] Now that your control plane is upgraded, please proceed with upgrading your kubelets if you haven't already done so.
+    ```
+
+1.  Manually upgrade your Software Defined Network (SDN).
```
Comment (timothysc) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: Manually upgrade your CNI...

Comment (timothysc) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: Not everyone uses and SDN.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md (hunk 3 above)
Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: this is something we decided on last cycle
cc @mauilion

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md (hunk 4 above)
Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md]: we are just copy-pasting from the old docs for 1.13, but i guess we can make the change here.

**Comment by neolit123** [comment]
closing in favor of: https://github.com/kubernetes/website/pull/13189
/close

**Pull request closed without merging**